ckan/plugins/interfaces.py

```diff
@@ -1618,6 +1618,10 @@ def get_resource_uploader(self):
         :param resource: resource dict
         :type resource: dictionary
 
+        Optional attributes:
```

Comment:
Can you expand this a bit more to make it more clear to people less familiar with how it works? Something like:

"Optionally, this method can set the following two attributes on the class instance so they are set in the resource object:"